the domd script is a huge hack already, so add a little more to handle ccache

--- a/util/domd
+++ b/util/domd
@@ -6,6 +6,7 @@
 shift
 if [ "$1" = "-MD" ]; then
     shift
+    [ "$1" = "ccache" ] && shift
     MAKEDEPEND=$1
     shift
 fi
